k9mail/src/main/java/com/fsck/k9/helper/Utility.java

```diff
@@ -111,21 +111,22 @@ public static boolean requiredFieldValid(TextView view) {
 
 
     public static boolean requiredFieldValid(Editable s) {
+        return s != null && requiredFieldValid(s.toString());
```

Comment:
To test presenter classes, is it needed to make all involved static methods in `Utility` non-static?